RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -1454,35 +1454,7 @@ boolean L13_towerNSTower()
 			abort("auto_towerBreak set to abort here.");
 		}
 		equipBaseline();
-		shrugAT($effect[Polka of Plenty]);
-		buffMaintain($effect[Disco Leer]);
-		buffMaintain($effect[Polka of Plenty]);
-		buffMaintain($effect[Cranberry Cordiality]);
-		buffMaintain($effect[Big Meat Big Prizes]);
-		buffMaintain($effect[Patent Avarice]);
-		buffMaintain($effect[Flapper Dancin\']);
-		buffMaintain($effect[Incredibly Well Lit]);
-		bat_formWolf();
-		if(auto_birdModifier("Meat Drop") > 0)
-		{
-			buffMaintain($effect[Blessing of the Bird]);
-		}
-		if(auto_favoriteBirdModifier("Meat Drop") > 0)
-		{
-			buffMaintain($effect[Blessing of Your Favorite Bird]);
-		}
-		if((get_property("sidequestArenaCompleted") == "fratboy") && !get_property("concertVisited").to_boolean() && (have_effect($effect[Winklered]) == 0))
-		{
-			cli_execute("concert 2");
-		}
-		
-		handleFamiliar("meat");
-		addToMaximize("200meat drop");
-		
-		if(my_class() == $class[Seal Clubber])
-		{
-			autoEquip($item[Meat Tenderizer is Murder]);
-		}
+		provideMeat(525.625, true, false);
```

Comment:
personal preference, but I'd round to 526 to keep numbers simple. Not a big deal either way